src/backend/termion.rs

```diff
@@ -137,11 +141,21 @@ where
     }
 
     fn get_cursor(&mut self) -> io::Result<(u16, u16)> {
-        termion::cursor::DetectCursorPos::cursor_pos(&mut self.writer).map(|(x, y)| (x - 1, y - 1))
+        match termion::cursor::DetectCursorPos::cursor_pos(&mut self.writer) {
+            Ok((x, y)) => {
+                let cursor = (x - 1, y - 1);
+                self.cursor = Some(cursor);
+                Ok(cursor)
+            }
+            Err(_) => self
+                .cursor
+                .ok_or_else(|| io::Error::other("No last known cursor position")),
+        }
     }
 
     fn set_cursor(&mut self, x: u16, y: u16) -> io::Result<()> {
```

Comment:
And perhaps doc comment here ("A cursor position set here will be used as a fallback...")